src/esp/gfx/Drawable.h

```diff
@@ -50,6 +50,20 @@ class Drawable : public Magnum::SceneGraph::Drawable3D {
    */
   uint64_t getDrawableId() { return drawableId_; }
 
+  /**
+   * @brief Return if this drawable object is selected in the editor (e.g.,
+   * viewer)
+   * @return the drawable object state, if it is selected
+   */
+  bool selected() { return selected_; }
```

Comment:
`getSelected`? `isSelected`? not exactly sure what's the naming scheme for bool getters here. Also

```suggestion
  bool selected() const { return selected_; }
```